app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -168,7 +168,7 @@ def set_pre_prov_vars
 
     unless %w[image_miq_request_new miq_template_miq_request_new].include?(params[:pressed])
       path_to_report = ManageIQ::UI::Classic::Engine.root.join("product", "views", provisioning_report).to_s
-      @view = MiqReport.new(YAML.safe_load(File.open(path_to_report), [Symbol]))
+      @view = MiqReport.new(YAML.load(File.open(path_to_report)))
```

Comment:
Maybe not for this PR, but this opens a file handle and then never closes it.  Perhaps instead use `YAML.load_file` or `MiqReport.new(File.open(path_to_report) { |f| YAML.load(f) })`

Reply:
yes, or `File.read`

Reply:
I'll do a followup PR

Reply:
I'll use `File.read`  in case we want to enumerate the possible classes and change this back to `safe_load` since there is no `YAML.safe_file_load`.

Reply:
makes sense, thanks :)

Reply:
Opened a PR... note, I found other instances of the same pattern:
https://github.com/ManageIQ/manageiq-ui-classic/pull/6597